src/app/infinite-scroller.ts

```diff
@@ -11,9 +11,23 @@ export class InfiniteScroller {
     private pageSize: number,
     private getPage: (page: number) => any[] | Promise<any>,
     private windowViewport: boolean,
-    private bufferSize: number = 50
+    private bufferSize: number = 50,
+    private padding: number = undefined
   ) {}
 
+  settingsWithoutPadding = {
+    bufferSize: this.bufferSize,
+    infinite: true,
+    minIndex: 0,
+    startIndex: 0,
+    windowViewport: this.windowViewport,
+  };
+
+  settingsWithPadding = {
+    ...this.settingsWithoutPadding,
+    padding: this.padding,
+  };
```

Comment:
Adding in the `padding` for the notifications page was causing the pagination to break, so I split up the settings in this way. I'm open to other options here if there are any cleaner ideas